configure

```diff
@@ -7,26 +7,26 @@
 ##  Copyright 2002 Lucumo   ##
 ##############################
 
-prefix='/usr/local/'
-bindir='$prefix/bin/'
-sbindir='$prefix/sbin/'
-etcdir='$prefix/etc/bitlbee/'
-mandir='$prefix/share/man/'
-datadir='$prefix/share/bitlbee/'
-config='/var/lib/bitlbee/'
-libdir='$prefix/lib/'
-plugindir='$prefix/lib/bitlbee/'
-includedir='$prefix/include/bitlbee/'
-systemdsystemunitdir=''
-sysusersdir=''
-libevent='/usr/'
-pidfile='/var/run/bitlbee.pid'
-ipcsocket=''
-pcdir='$prefix/lib/pkgconfig'
+prefix="/usr/local/"
+bindir="$prefix/bin/"
+sbindir="$prefix/sbin/"
+etcdir="$prefix/etc/bitlbee/"
+mandir="$prefix/share/man/"
+datadir="$prefix/share/bitlbee/"
+config="/var/lib/bitlbee/"
+libdir="$prefix/lib/"
+plugindir="$prefix/lib/bitlbee/"
+includedir="$prefix/include/bitlbee/"
+systemdsystemunitdir=""
+sysusersdir=""
+libevent="/usr/"
+pidfile="/var/run/bitlbee.pid"
+ipcsocket=""
+pcdir="$prefix/lib/pkgconfig"
```

Comment:
This change is quite problematic. `$prefix` is substituted immediately because you have changed single quotes to double quotes but configuration options are parsed later. This results in this:
```
$ ./configure --prefix=/usr --help
BitlBee configure

Usage: ./configure [OPTIONS]

Option          Description                             Default

--prefix=...    Directories to put files in             /usr
--bindir=...                                            /usr/local//bin/
--sbindir=...                                           /usr/local//sbin/
--etcdir=...                                            /usr/local//etc/bitlbee/
--mandir=...                                            /usr/local//share/man/
--datadir=...                                           /usr/local//share/bitlbee/
--plugindir=...                                         /usr/local//lib/bitlbee/
--systemdsystemunitdir=...
--sysusersdir
--pidfile=...                                           /var/run/bitlbee.pid
--config=...                                            /var/lib/bitlbee/
```
Then, the `Makefile.settings` looks like this.
```make
## BitlBee settings, generated by configure

# ./configure --prefix=/usr

PREFIX=/usr
BINDIR=/usr/local/bin/
SBINDIR=/usr/local/sbin/
ETCDIR=/usr/local/etc/bitlbee/
MANDIR=/usr/local/share/man/
DATADIR=/usr/local/share/bitlbee/
PLUGINDIR=/usr/local/lib/bitlbee/
CONFIG=/var/lib/bitlbee/
LIBDIR=/usr/local/lib/
INCLUDEDIR=/usr/local/include/bitlbee/
PCDIR=/usr/local/lib/pkgconfig
```
but I really don't want it to install files to `/usr/local` :) The substitution is postponed with single quotes, for example it worked like this in commit 8ca172fc:
```
$ ./configure --prefix=/usr --help
BitlBee configure

Usage: ./configure [OPTIONS]

Option          Description                             Default

--prefix=...    Directories to put files in             /usr
--bindir=...                                            $prefix/bin/
--sbindir=...                                           $prefix/sbin/
--etcdir=...                                            $prefix/etc/bitlbee/
--mandir=...                                            $prefix/share/man/
--datadir=...                                           $prefix/share/bitlbee/
--plugindir=...                                         $prefix/lib/bitlbee/
--systemdsystemunitdir=...
--sysusersdir
--pidfile=...                                           /var/run/bitlbee.pid
--config=...                                            /var/lib/bitlbee/
```
and `Makefile.settings`:
```make
## BitlBee settings, generated by configure

# ./configure --prefix=/usr

PREFIX=/usr
BINDIR=/usr/bin/
SBINDIR=/usr/sbin/
ETCDIR=/usr/etc/bitlbee/
MANDIR=/usr/share/man/
DATADIR=/usr/share/bitlbee/
PLUGINDIR=/usr/lib/bitlbee/
CONFIG=/var/lib/bitlbee/
LIBDIR=/usr/lib/
INCLUDEDIR=/usr/include/bitlbee/
PCDIR=/usr/lib/pkgconfig
```

Reply:
Yeah, this is the third unintentional regression from this PR. I think I'll roll it back for the moment.